### ISSUE TITLE
inkscape-open-symbols license

### ISSUE THREAD

**Issue by legrostdg** [Xaviju/inkscape-open-symbols#61]
Could you please update your license to GPLv2+ (instead of GPLv2 only)?
GPLv2 is not compatible with the other licenses.

**Comment by legrostdg** [comment]
you can use https://github.com/Automattic/genericons-neue/blob/master/COPYING.md as a basis.

**Comment by legrostdg** [comment]
note that I need this to finish the Debian package.

**Comment by PanderMusubi** [comment]
Perhaps unneeded to ask, this will be a noarch package, right?

**Comment by Xaviju** [comment]
Yes, I've been discussing with a colleague to understand better if I could change the license to a less restrictive one (MIT), while keeping the license of the current symbols. Actually, there is no much to license in this repo if the symbols remain with their licenses.
After the discussion, I'll probably relicense it as MIT adding a note to the contribution page to warn that all symbols will keep their original licenses.
I'll try to do it today.

**Comment by legrostdg** [comment]
You can't be MIT if you distribute GPL files. Maybe it could work there but I'm not sure...

**Comment by legrostdg** [comment]
@PanderMusubi the package will be available for all Debian release architectures. I'll upload a source package.

**Comment by PanderMusubi** [comment]
Please, notify also in https://bugs.launchpad.net/ubuntu/+bug/1464270 where to get it.

**Comment by niwinz** [comment]
@legrostdg it depends on the package, when you are distributing a source of some application with some kind of modification or as is, you should respect as is the package and release your changes using the same license (in this case gpl). But, when the package like this that contains just a collection of icons, where each collection has its own license, the entire package license does not means nothing. Or in other words, the package level license is licensing everything except the collections of icons (in this concrete case the python script and gulp file).

In summary: each collection has its own license and any one using any of them should respect its license, the build tools  and other stuff unrelated to collections is licensed with the package level license (LICENSE file on the root of the repository). 

And in this case, as @Xaviju is the only one owner of that files, he can change it to any license he wants. Because that change does not affect the collections licenses in any manner.

**Comment by PanderMusubi** [comment]
Ah, you can ask inkscape package manager to add as suggested package inkscape-open-symbols

**Comment by legrostdg** [comment]
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=850821
Here is the discussion about the package (and the license problem).

**Comment by niwinz** [comment]
@Xaviju will change the package level license to MIT, but independently of that action, there are no compatibility issues because the package level license does not affect the collections issues, **because each one has its own license and we all should respect them as-is**.

**Comment by legrostdg** [comment]
@niwinz if I get what you say correctly, the package can be under whatever (DFSG compatible) license, right?
Do you think the Debian package could be in whatever (DFSG compatible) license, too?

**Comment by niwinz** [comment]
What DFSG means?

**Comment by legrostdg** [comment]
Debian Free Software Guidelines https://en.wikipedia.org/wiki/DFSG.

**Comment by niwinz** [comment]
Just use the package level license as debian package license but indicating somewhere that the package contains content with multiple different licenses and the user should be aware that using one or other collection implies to be subject to the license of the used icons collections.

However, please wait until @Xaviju changes the package level license to one more permissive such as MIT

**Comment by Moini** [comment]
Just in case anyone thinks of asking contributors if they're okay with a licence change: I'm okay with any licence that makes it easy for users to use, modify and distribute this collection.

**Comment by niwinz** [comment]
Thanks @Moini , 

The license change will affect very small amount of files, just concretely the gulpfile and the generate.py file that a re 100% owned by @Xaviju (they does not have any modifications by the contributors).

If you have contributed to some collection that has its own license, it will be untouched, the collections license will not to be changed.

**Comment by Xaviju** [comment]
Thanks @niwinz, @legrostdg, @PanderMusubi and @Moini! I think is the best idea!
I'm all about making it easy for the users. It will probably help future enhancements of the building tools.

@Moini I brought the details of the discussion here to listen your opinions as well. :)

**Comment by Moini** [comment]
Oh, I wasn't thinking I was being left out. I was just trying to make it easy, in case it could be important.

**Comment by Xaviju** [comment]
@legrostdg do you need anything else to  to finish the Debian package?

**Comment by legrostdg** [comment]
@Xaviju Could you make a 1.1 release?

**Comment by Xaviju** [comment]
Sure. No problem

**Comment by Xaviju** [comment]
https://github.com/Xaviju/inkscape-open-symbols/releases/tag/v1.1
